api/v1beta1/common_types.go

```diff
@@ -117,6 +121,11 @@ const (
 const (
 	// TemplateSuffix is the object kind suffix used by template types.
 	TemplateSuffix = "Template"
+
+	// ClusterTopologyAdoptionTypeManual value is set on ClusterTopologyAdoptionAnnotation when
+	// a user wants to experiment manual adaption from an unmanaged cluster to a managed cluster with ClusterClass
+	// manually and the user understands are no support guarantees for manual adaption.
+	ClusterTopologyAdoptionTypeManual = "Manual"
```

Comment:
As far as I know, we are not foreseeing different types of adoption at this stage, so I'm wondering if we can just rely on the annotation exists or not (without relying on specific values)

Reply:
I think it would be good for now to only check if it exists. Otherwise it kind of implies that there is or will be a non-manual adoption.

Reply:
agreed that relying on the annotation exists or not is good enough for the current use case.  the reasons I had it as "Manual" was to kinda make it more future proof so that in case there are different types at adoptions. the existing users would not need to change their codebase; otherwise, the exist users may need to change the random values they set for the adoption annotation key.  

I can also change the logic to check only the existence of annotation as well if the community sees no benefits in this.

Reply:
+1 to remove this value and just rely on the annotation to be present